field/src/field.rs

```diff
@@ -121,6 +121,19 @@ pub trait AbstractField:
         }
     }
 
+    fn packed_powers<P: PackedField<Scalar = Self>>(&self, start: Self) -> PackedPowers<Self, P> {
```

Comment:
Ideally the API would match `powers`, maybe having users instantiate `PackedPowers` if they need a `start`, or maybe separate methods like `shifted_powers` for both would be cleaner. I think we can leave it for later though.

Reply:
I added them here